contracts/test/NativeTokenHomeTests.t.sol

```diff
@@ -18,7 +18,7 @@ contract NativeTokenHomeTest is NativeTokenBridgeTest, TokenHomeTest {
     using SafeERC20 for IERC20;
 
     NativeTokenHome public app;
-    IWrappedNativeToken public mockWrappedToken;
+    IWrappedNativeToken public wavax;
```

Comment:
Super nit: Seems like this name change is unnecessary, but fine either way.

Reply:
decided to go ahead and remove the mock for naming